docs/definition.rst

```diff
@@ -4,14 +4,41 @@ Execution Environment Definition
 The execution environment (EE) definition file supports multiple versions.
 
   * Version 1: Supported by all ``ansible-builder`` versions.
-  * Version 2: Supported by ``ansible-builder`` versions ``1.2`` and higher.
+  * Version 2: Supported by ``ansible-builder`` versions ``2.0`` and higher.
 
 :ref:`Version 2 <version-2>` adds the capability to optionally use and verify
 signed container images. This feature is only supported with the ``podman``
 container runtime.
 
 If the EE file does not specify a version, version 1 will be assumed.
 
+.. _build-phases:
+
+Build Phases
+------------
+
+The EE configuration file controls how the final image is created. Understanding
+how Ansible Builder works to create the final image can help you better
+understand how to create your EE file.
+
+Ansible Builder creates an execution environment image by splitting its work
+into multiple intermediary images/stages:
+
+  * Base image: This image is used as the basis for (almost) all subsequent phases.
+    The exception is the builder image which will use the builder image specified
```

Comment:
(unless we kill that feature in favor of "bring your own build stage" :laughing: )

Reply:
Maybe we should come up with a different name for `base`; it can be confusing whether we're talking about the actual (now vanilla) base image or our intermediate base with Python/core/etc installed...

Reply:
Good point, and it made for some clumsy sentences. `primary` or `initial` immediately come to mind as potential replacements.